MREUnityRuntime/MREUnityRuntimeLib/Core/Physics/JitterBuffer.cs

```diff
@@ -232,235 +320,577 @@ public void add(float bufferTime, float meanBufferTime, float targetBufferTime,
 			private DebugStats _debugStats = new DebugStats();
 #endif
 
-			public SourceInfo(Guid id)
+#endregion
+
+#region Time Management
+
+		/// <summary>
+		/// Time precision is 1 millisecond.
+		/// </summary>
+		private const float _timePrecision = 0.001f;
+
+		/// <summary>
+		/// todo: add comment
+		/// </summary>
+		private float _speedUpCoef = 0.2f;
+
+		/// <summary>
+		/// todo: add comment
```

Comment:
>/// todo: add comment [](start = 2, length = 21)

Add comment :)

Reply:
deleted

---
In reply to: [450354994](https://github.com/microsoft/mixed-reality-extension-unity/pull/210#discussion_r450354994) [](ancestors = 450354994)